build.gradle

```
@@ -103,10 +107,17 @@ task rtd_pdf(type:Exec){
 }
 
 task clean_conf(type:Delete) {
-    delete 'conf.py',
-    'libs',
-	'_static',
-	'logo.png'
+    if("${projectDir.parent.contains("RTD-UM")}"){
```

```suggestion
    if("${projectDir.parent.contains("RTD_UM")}" || "${projectDir.parent.contains("RTD-UM")}"){
```
Existen ciertos manuales cuyo nombre es RTD_UM en lugar de RTD-UM. Lo más correcto sería corregir estos manuales para que su codificación sea la correcta (no nos tocaría a nosotros corregirlos sino a los responsables de dichos manuales), pero, mientras esta corrección sucede se puede controlar este problema añadiendo este caso en la condición del if